units/patch_intent.go

```diff
@@ -534,6 +534,14 @@ func (j *patchIntentProcessor) createGitHubMergeSubscription(ctx context.Context
 	}))
 	// If we don't find any rules, send the default.
 	if len(rules) == 0 {
+		grip.Debug(message.Fields{
```

Comment:
non blocking: 
you might want to log p.Project as well, it makes it a bit easier to search for things and figure out who to talk to (so that you can search for the project page and don't need to query the db)

Reply:
oh good point thanks!